edv4ldez/react-apprenticeship-capstone1
# Capston project 1 Final Deliverable

## Description
``` 
Final Deliverable
Entrega del capston project 1.
- Al hacer login puedes seleccionar los videos como favoritos dando click en la estrella y los puedes ver en la sección de favoritos.
- Tengo que corregir un bug relacionado con el search bar, al buscar algo en la ruta videos/videoId la aplicación deja de funcionar.
- Debo de realizar más testing.
- Falta guardar los videos favoritos en el localstorage. 

```
## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 
- Las test se encuentran en el directorio tests 

Add any information regarding the PR that the reviewers should know, if necessary, for example "This is only a fix PR related the past PR (PR link)" 
```
## What is the deliverable?
``` Final Deliverable
Number of the deliverable and the tittle of this one. 
``` Final Deliverable
## Screenshots and links
```https://edv4ldez.github.io/react-apprenticeship-capstone1/

```
## Checklist
- [] I added the necessary documentation, if appropriate.
